src/main/resources/application.yaml

```diff
@@ -53,6 +53,12 @@ boober:
         suffix: ".${openshift.cluster}.paas.skead.no"
     webseal:
         suffix: ".amutv.skead.no"
+    productionlevel:
```

Comment:
Just use the value directly here and override in AuroraConfig for test/prod.

Reply:
Fixed